lms/envs/common.py

```diff
@@ -1019,6 +1019,16 @@
     # .. toggle_target_removal_date: None
     # .. toggle_tickets: 'https://openedx.atlassian.net/browse/MST-1458'
     'ENABLE_CERTIFICATES_IDV_REQUIREMENT': False,
+
+    # .. toggle_name: FEATURES['DISABLE_AUTO_ENROLL_AFTER_ENROLLMENT_END']
+    # .. toggle_implementation: DjangoSetting
+    # .. toggle_default: False
+    # .. toggle_description: Set to True to disable auto enrollment of user to a course
+    # .. if user is registering after the enrollment end date
+    # .. toggle_use_cases: open_edx
+    # .. toggle_creation_date: 2022-05-23
+    # .. toggle_tickets: 'https://github.com/edx/edx-platform/pull/29538'
+    'DISABLE_AUTO_ENROLL_AFTER_ENROLLMENT_END': False,
```

Comment:
See my other comments - we will probably need a more accurate name here. Something with "disabling enrollments for students invited into the course when the course enrollment is not open", but more user-friendly. Maybe `DISABLE_ALLOWED_ENROLLMENTS_IF_ENROLLMENT_CLOSED` (though I'm not a huge fan of it).

Reply:
Changed